crates/gitbutler-repo/src/lib.rs

```diff
@@ -16,6 +16,8 @@ pub use config::Config;
 
 pub mod temporary_workdir;
 
+pub mod logging;
```

Comment:
probably nicer to re-export `LogUntil` from the root of `gitbutler_repo` and keep the logging mod private, for better discoverability and imports readability